src/Grok.Net/Grok.cs

```diff
@@ -158,10 +158,23 @@ private void ProcessPatternLine(string line)
             }
 
             string[] strArray = line.Split(new[] { ' ' }, 2);
+            if (strArray.Length != 2)
+            {
+                throw new FormatException(String.Format("Custom pattern was not in a correct form"));
```

Comment:
Could you please explain why do you use `String.Format` here?

Reply:
My bad, I used it before to include the wrong pattern in the exception message but then I removed it and forgot to change this.